packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -325,6 +342,35 @@ class CameraController extends ValueNotifier<CameraValue> {
     await CameraPlatform.instance.prepareForVideoRecording();
   }
 
+  /// Pauses the current camera preview
+  Future<void> pausePreview() async {
+    if (value.isPreviewPaused) {
+      return;
+    }
+    try {
+      await CameraPlatform.instance.pausePreview(_cameraId);
+      value = value.copyWith(
+          isPreviewPaused: true,
+          previewPauseOrientation: Optional.of(this.value.deviceOrientation));
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
+
+  /// Resumes the current camera preview
+  Future<void> resumePreview() async {
+    if (!value.isPreviewPaused) {
+      return;
+    }
+    try {
+      await CameraPlatform.instance.resumePreview(_cameraId);
+      value = value.copyWith(
+          isPreviewPaused: false, previewPauseOrientation: Optional.absent());
+    } on PlatformException catch (e) {
+      throw CameraException(e.code, e.message);
+    }
+  }
```

Comment:
This should also be covered by tests. One when the method call succeeds and one to verify a `PlatformException` is handled correctly.